genie_registry/clinical.py

```diff
@@ -823,6 +850,14 @@ def _validate(self, clinicaldf):
         )
         total_error.write(error)
 
+        # CHECK: YEAR DEATH against YEAR CONTACT
+        haveColumn = process_functions.checkColExist(
+            clinicaldf, ["YEAR_DEATH", "YEAR_CONTACT"]
+        )
+        if haveColumn:
+            error = _check_year_death_validity(clinicaldf=clinicaldf)
+            total_error.write(error)
+
```

Comment:
A stretch goal outside of the explicit scope of this change would be to split out these other items into their own functions. Is there a tech debt jira that already exists, or could one be created to address this item?

Cc: @rxu17

Reply:
Yes we have a tech debt jira for this. The goal is to eventually have a validation function for each validation rule for each fileformat.